index.js

```diff
@@ -40,28 +40,22 @@ const storage = {
   
   getText: () => 
     new Promise(function(resolve) {
-      chrome.storage.sync.get(null, items => {
+      chrome.storage.largeSync.get(null, items => {
         resolve(items);
       });
     }).then(storage.joinText),
   
   splitText: text => {
-    let splittedText = {};
-    storage.splitKeys().forEach(i => {
-      splittedText['text' + i] = text.substring(i * storage.QUOTA_BYTES_PER_ITEM, (i + 1) * storage.QUOTA_BYTES_PER_ITEM);
-    });
-    return splittedText;
+    return {text};
```

Comment:
`splitText` is not the best name for this method anymore. Maybe something like `textToObject`?

Reply:
yep!